airbyte-integrations/connectors/source-facebook-marketing/acceptance-test-config.yml

```diff
@@ -14,13 +14,17 @@ tests:
   basic_read:
     - config_path: "secrets/config.json"
       configured_catalog_path: "integration_tests/configured_catalog.json"
-      validate_output_from_all_streams: yes
+      empty_streams:
```

Comment:
can you create a ticket for populating these streams? we can have karina create ad campaigns

Reply:
In [this](https://github.com/airbytehq/airbyte/issues/4827#issuecomment-886981239) comment @keu provided config and with that config we have all the data. So right now it's a little bit confusing and it's not clear which config to use:
- the one which is currently in LastPass (in this case we really need to create a ticket for populating these streams)
- config which sent Eugene